.github/workflows/test_build.yml

```diff
@@ -134,7 +134,8 @@ jobs:
             -DCMAKE_PREFIX_PATH="`brew --prefix`;`brew --prefix libffi`" \
             -DCMAKE_INSTALL_PREFIX=/usr \
             -DWITH_PYTHON=ON \
-            --debug-trycompile
+            --debug-trycompile \
+            -DWITH_FFI=OFF
```

Comment:
Why does CMake build without FFI?

Reply:
This was from a temporary commit to check and make sure that the various builds all worked when disabling libffi.  They did, so I'll remove it here in a moment.

Reply:
Ah I see.